docs/buildah.md

```diff
@@ -27,6 +27,21 @@ Print debugging information
 **--default-mounts-file**
 
 Path to default mounts file (default path: "/usr/share/containers/mounts.conf")
+Alternate path used `/etc/containers/mounts.conf` if user created it.
+
+The mounts.conf file(s) specify volume mount directories that are automatically mounted inside containers.
+The processes running in the container during a `buildah run` or a `buildah build-using-dockerfile` can use
+this content but the content does not get committed to the final image.
+
+Usually these directories
+are used for passing secrets or credentials required by the package software to access remote package repositories.
+When you have a mounts.conf with `/usr/share/rhel/secrets:/run/secrets`, the content of /usr/share/rhel/secrets directory
+is mounted on /run/secrets inside the container.  This allows Red Hat Enterprise Linux subscriptions from the host to be used
+inside of the container.
+
+The format of the mounts.conf is /SRC:/DEST.  Users can create an `/etc/containers/mounts.conf`, to specify their
```

Comment:
I assume I can specify more than one mount.
I think it's worth mentioning how they are separated (by a comma, a space, a new-line)

Reply:
Updated